client/components/breadcrumb/index.tsx

```diff
@@ -114,14 +115,26 @@ const Breadcrumb: React.FunctionComponent< Props > = ( props ) => {
 		);
 	}
 
+	function gotoLink( href: string | undefined ) {
+		if ( href !== undefined ) {
+			page( href );
+		}
+	}
+
 	if ( items.length > 1 ) {
 		return (
 			<StyledUl>
 				{ items.map( ( item: { href?: string; label: string }, index: Key ) => (
 					<StyledLi key={ index }>
 						{ index !== 0 && <StyledGridicon icon="chevron-right" size={ 14 } /> }
 						{ item.href && index !== items.length - 1 ? (
-							<a href={ item.href }>{ item.label }</a>
+							<button
```

Comment:
We can use a `<span role="link">` here like these [other ones](https://github.com/Automattic/wp-calypso/blob/9a0be7e684cb536f864210848a753d8cab6efb96/client/my-sites/plugins/plugins-browser-item/index.jsx#L212-L220).

Would really prefer to figure out why `<a>` tags aren't navigating though.

Reply:
> Would really prefer to figure out why <a> tags aren't navigating though.

I'd like +1 this as there're ongoing efforts (#66169 and #66344) to make this section available to logged-out users and make it SEO-friendly. In this case, we'll still need to figure out how to make the `<a>` tags work properly.

Reply:
Lets discus in the issue over [here](https://github.com/Automattic/wp-calypso/issues/66795#issuecomment-1223860562).